management-api-server/src/main/java/com/datastax/mgmtapi/resources/models/FeatureSet.java

```diff
@@ -23,6 +23,8 @@ public enum Feature {
     ASYNC_SCRUB_TASK("async_scrub_task"),
     ASYNC_MOVE_TASK("async_move_task"),
     ASYNC_GC_TASK("async_gc_task"),
+    RELOAD_INTERNODE_TRUSTSTORE("reload_internode_truststore"),
+    GET_INTERNODE_TRUSTSTORE("get_internode_truststore"),
```

Comment:
This should probably be removed, only the reload_internode_truststore is valid.

Reply:
Good catch

Reply:
Done.